3rdparty/libgcov/README.md

```diff
@@ -0,0 +1,13 @@
+# libgcov
```

Comment:
Should you include the license at the top of this file?

Reply:
I guess README.md file is an exception?